test/e2e/nightly/pvc_stress_fio/pvc_stress_fio_test.go

```diff
@@ -169,20 +201,48 @@ func TestPVCStress(t *testing.T) {
 	RunSpecs(t, "PVC Stress Test Suite")
 }
 
-var _ = Describe("Mayastor PVC Stress test with fio", func() {
-	It("should stress test use of PVCs provisioned over iSCSI and NVMe-of", func() {
-		stressTestPVC()
+var _ = Describe("Mayastor PVC Stress test", func() {
+	It("should stress test creation and deletion of PVCs provisioned over iSCSI and NVMe-of", func() {
+		stressTestPVC(cdIterations, false)
 	})
-})
+
+	It("should stress test creation and deletion of PVCs provisioned over iSCSI and NVMe-of", func() {
+		stressTestPVC(crudIterations, true)
+	})})
 
 var _ = BeforeSuite(func(done Done) {
-	logf.SetLogger(zap.LoggerTo(GinkgoWriter, true))
+	logf.SetLogger(zap.New(zap.UseDevMode(true), zap.WriteTo(GinkgoWriter)))
 
 	Cmn.SetupTestEnv()
+	tmp := os.Getenv("pvc_stress_cd_cycles")
+	if len(tmp) != 0 {
+		var err error
+		cdIterations, err = strconv.Atoi(tmp)
+		Expect(err).NotTo(HaveOccurred())
+		logf.Log.Info("Cycle count changed by environment ", "Create/Delete", cdIterations)
+	}
+
+	tmp = os.Getenv("pvc_stress_crud_cycles")
+	if len(tmp) != 0 {
+		var err error
+		crudIterations, err = strconv.Atoi(tmp)
+		Expect(err).NotTo(HaveOccurred())
+		logf.Log.Info("Cycle count changed by environment", "Create/Read/Update/Delete", crudIterations)
+	}
+	logf.Log.Info("Number of cycles are", "Create/Delete", cdIterations, "Create/Read/Update/Delete", crudIterations)
+
 	close(done)
 }, 60)
 
 var _ = AfterSuite(func() {
+	// Cleanup resources leftover in the event of failure.
+	for _, pod := range podNames {
+		_ = Cmn.DeletePod(pod)
```

Comment:
What happens if the delete pod fails? Could you please make sure that if the cleanup fails whole run fails so we can detect an error in the job? Given that we depend on ending in clean state for now this will help us debug why we don't have one.

Reply:
Added check for error.